frontend/src/views/c-authorship.vue

```diff
@@ -437,7 +442,7 @@ export default {
       }
       if (repoCache.length === 2) {
         const toRemove = repoCache.shift();
-        if (toRemove !== this.info.repo) {
+        if (toRemove !== this.info.repo && toRemove !== undefined) {
```

Comment:
2 minor points here.

1. you can simply use `if (toRemove)` to check if it is undefined
2. It is always a better practice to check for undefined first

Reply:
Thanks for the pointers, will make the changes & keep this in mind in the future!